src/main/java/ua/fan/hw4/Task2_3.java

```diff
@@ -0,0 +1,65 @@
+package ua.fan.hw4;
+
+import java.util.Arrays;
+
+public class Task2_3 {
```

Comment:
[6, 10, 4, 6, 8, 9, 9, 10, 4, 3, 10, 2, 10, 10, 5, 6, 1, 1, 7, 10, 1, 1, 7, 2, 2, 7, 7, 6, 5, 8, 9, 2, 3, 9, 8, 10, 8, 10, 3, 9, 4, 1, 6, 9, 9, 10, 2, 2, 1, 4, 8, 3, 10, 10, 1, 7, 2, 5, 6, 10, 8, 1, 6, 8, 5, 5, 6, 8, 5, 1, 2, 1, 7, 4, 6, 2, 4, 10, 4, 4, 2, 5, 10, 8, 5, 5, 3, 5, 1, 4, 8, 4, 6, 6, 10, 3, 8, 9, 6, 5, 2, 3, 8, 9, 9, 5, 3, 2, 3, 8, 4, 9, 10, 9, 8, 8, 7, 9, 6, 7, 7, 7, 5, 8, 4, 5, 8, 1, 4, 3, 4, 6, 10, 4, 1, 8, 6, 1, 10, 10, 8, 10, 3, 2, 6, 9, 10, 6, 8, 5, 2, 4, 9, 5, 10, 10, 7, 1, 10, 8, 8, 5, 7, 4, 6, 3, 3, 1, 2, 6, 6, 3, 5, 2, 8, 5, 1, 10, 2, 1, 3, 8, 9, 3, 10, 9, 6, 1, 2, 2, 1, 2, 10, 6, 6, 5, 7, 5, 10, 8, 7, 4, 7, 6, 1, 6, 2, 5, 4, 10, 1, 4, 6, 1, 3, 5, 3, 4, 4, 5, 3, 8, 2, 6, 2, 6, 2, 5, 6, 3, 8, 6, 8, 6, 10, 9, 5, 4, 1, 2, 3, 8, 5, 10, 3, 1, 10, 6, 6, 9, 3, 4, 6, 5, 2, 4, 8, 5, 7, 8, 10, 9, 10, 10, 3, 2, 5, 8, 3, 2, 3, 8, 4, 2, 5, 2, 7, 7, 3, 7, 4, 4, 9, 4, 6, 1, 10, 10, 10, 3, 8, 5, 2, 9, 9, 4, 1, 4, 5, 3, 5, 7, 1, 8, 3, 2, 4, 6, 6, 1, 2, 2, 2, 8, 10, 9, 1, 5, 5, 9, 6, 5, 6, 10, 3, 10, 6, 4, 1, 6, 6, 7, 3, 3, 10, 3, 8, 9, 8, 6, 2, 10, 1, 5, 7, 4, 3, 6, 5, 4, 8, 8, 2, 8, 7, 7, 9, 1, 4, 8, 5, 5, 8, 8, 3, 1, 4, 1, 8, 1, 8, 1, 2, 7, 5, 7, 10, 8, 2, 6, 1, 1, 8, 6, 10, 9, 2, 5, 8, 10, 9, 5, 1, 6, 1, 3, 8, 3, 4, 4, 3, 1, 8, 5, 2, 10, 8, 4, 10, 7, 10, 7, 3, 9, 5, 7, 3, 7, 6, 2, 2, 5, 10, 7, 9, 1, 3, 5, 6, 8, 6, 4, 2, 3, 2, 8, 9, 1, 3, 9, 2, 9, 1, 9, 9, 7, 2, 6, 2, 3, 5, 9, 10, 8, 8, 1, 10, 7, 1, 1, 8, 5, 2, 6, 3, 1, 3, 4, 4, 10, 3, 10, 7, 10, 8, 6, 6, 1, 7, 8, 5, 3, 10, 2, 5, 4, 10, 5, 5, 9, 10, 5, 1, 10, 2, 9, 4, 6, 1, 4, 9, 10, 8, 8, 6, 3, 5, 5, 5, 1, 3, 7, 7, 4, 1, 7, 10, 5, 2, 3, 4, 4, 1, 3, 8, 5, 7, 10, 5, 8, 1, 10, 10, 9, 5, 7, 1, 2, 9, 8, 6, 3, 9, 1, 3, 6, 5, 4, 10, 5, 5, 4, 7, 4, 1, 7, 10, 9, 2, 6, 3, 5, 10, 5, 2, 4, 1, 9, 7, 7, 4, 4, 9, 7, 9, 6, 3, 4, 1, 8, 7, 10, 7, 5, 5, 5, 4, 7, 1, 7, 4, 4, 8, 4, 10, 9, 3, 10, 3, 9, 9, 2, 2, 5, 7, 10, 10, 9, 3, 3, 9, 7, 3, 10, 7, 2, 8, 9, 6, 2, 1, 9, 1, 4, 10, 7, 10, 5, 3, 7, 2, 4, 4, 4, 2, 10, 6, 7, 1, 1, 7, 4, 9, 10, 10, 2, 4, 8, 6, 9, 10, 1, 10, 7, 5, 1, 9, 7, 9, 10, 9, 1, 5, 3, 10, 8, 4, 4, 4, 8, 5, 8, 7, 10, 1, 8, 1, 4, 10, 3, 7, 5, 9, 7, 3, 3, 2, 10, 4, 9, 9, 7, 4, 1, 1, 5, 8, 1, 9, 1, 7, 5, 1, 1, 4, 10, 10, 6, 3, 9, 5, 4, 3, 9, 3, 7, 2, 3, 2, 10, 1, 10, 2, 10, 2, 3, 4, 5, 5, 10, 9, 9, 8, 4, 5, 8, 6, 10, 5, 4, 3, 1, 6, 9, 1, 10, 4, 4, 6, 7, 10, 9, 9, 8, 4, 10, 7, 10, 4, 4, 7, 2, 4, 8, 2, 7, 4, 2, 3, 10, 7, 8, 6, 1, 2, 4, 10, 3, 10, 9, 1, 5, 2, 10, 5, 3, 5, 8, 10, 6, 5, 3, 1, 7, 2, 1, 9, 7, 3, 8, 3, 4, 10, 1, 6, 4, 3, 10, 5, 10, 3, 2, 6, 9, 6, 9, 2, 3, 8, 4, 5, 6, 2, 7, 7, 5, 8, 7, 4, 10, 5, 5, 8, 7, 7, 5, 10, 8, 7, 8, 5, 9, 8, 6, 10, 2, 7, 2, 1, 1, 9, 4, 8, 3, 5, 4, 3, 1, 2, 6, 7, 9, 8, 6, 5, 2, 8, 8, 2, 1, 5, 1, 7, 10, 7, 3, 5, 9, 4, 4, 5, 4, 6, 7, 9, 5, 8, 4, 10, 7, 6, 9, 2, 4, 7, 8, 2, 6, 6, 3, 4, 10, 4, 9, 5, 2, 2, 7, 2, 10, 5, 9, 1, 5, 3, 3, 9, 5, 9, 1, 8, 4, 2, 2, 7, 3, 3, 8, 8, 4, 1, 10, 6, 6, 6, 9, 3, 6, 8, 9, 1, 9, 3, 9, 3, 1, 3, 7, 4, 1, 7, 10, 10, 9, 3, 10, 10, 2, 8, 4, 7, 7, 6, 7, 8, 1, 5, 10, 6, 8, 3, 10, 9, 3, 9, 8, 5, 4, 5, 6, 8, 7, 2, 5, 5, 10, 6, 10, 9, 4, 2, 8, 4, 8, 10, 8, 7, 9, 2, 7]
There are: 
392 prime and 511 composite numbers.